src/Sql/Sql/Instance Pools/Cmdlet/GetAzureSqlInstancePool.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Sql.Instance_Pools.Model;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+
+namespace Microsoft.Azure.Commands.Sql.Instance_Pools.Cmdlet
+{
+    /// <summary>
+    /// Defines the Get-AzRmSqlInstancePool cmdlet
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlInstancePool")]
+    [OutputType(typeof(AzureSqlInstancePoolModel))]
+    public class GetAzureSqlInstancePool : InstancePoolCmdletBase
+    {
+        /// <summary>
+        /// Parameter setss
+        /// </summary>
+        private const string ListBySubOrResourceGroupParameterSet = "ListBySubOrResourceGroupParameterSet";
+        private const string GetInstancePoolDefaultParameterSet = "ListByInstancePoolDefaultsParameterSet";
+        private const string GetInstancePoolByInstancePoolResourceIdentifierParameterSet = "GetInstancePoolByInstancePoolResourceIdentifierParameterSet";
+
+        /// <summary>
+        /// Gets or sets the resource group name
+        /// </summary>
+        [Parameter(
+            ParameterSetName = ListBySubOrResourceGroupParameterSet,
+            Mandatory = false,
+            Position = 0,
```

Comment:
@johnpaulkee please remove the position attribute from all _optional_ parameters

This comment applies to all instances of this in this PR.